docs/source/overview/deployment.rst

```diff
@@ -0,0 +1,183 @@
+    $ git status
+
+    $ git add policy.yml
+    $ git commit -m 'init my first policy'
+    $ git push -u origin master
+
+Once you've pushed your changes you should be able to see your new changes inside
+of Github. Congratulations, you're now ready to start automatically validating and
+testing your policies!
+
+.. _continuous_integration_of_policies:
+
+Continuous Integration of Policies
+----------------------------------
+
+Next, enable a CI webhook back to your CI system of choice when pull requests
+targeting your master branch are opened or updated. This allows us to constantly
+test and validate the policies that are being modified. You can use the tool
+c7n_policystream to generate diffs between the current master version and the
+proposed change in the pull request.
+
+In this example, we will be using Microsoft Azure Devops Pipelines.
+
+First, navigate to https://azure.microsoft.com/en-us/services/devops/pipelines/ and
+click the "Start pipelines free with Github" button and follow the flow to connect
+your Github account with Devops Pipelines.
+
+Next click on the Piplines section in the left hand side of the sidebar and connect
+with Github. Once the pipeline is setup, we can add the following azure devops
+configuration to our repo:
+
+.. code-block:: yaml
+
+    trigger:
+    - master
+    
+    jobs: 
+      - job: 'Validate'
+        pool:
+          vmImage: 'Ubuntu-16.04'
+        strategy:
+          matrix:
+            Python37:
+              python.version: '3.7'
+              TOXENV: py37
+        steps:
+          - checkout: self
+          - task: UsePythonVersion@0
+            displayName: "Set Python Version"
+            inputs:
+              versionSpec: '$(python.version)'
+              architecture: 'x64'
+          - script: pip install --upgrade pip
+            displayName: Upgrade pip
+          - script: pip install c7n
+            displayName: Install custodian
+          - script: custodian validate policy.yml
+            displayName: Validate policy file
+
+This configuration will install Cloud Custodian and validate the policy.yml file
+that we created in the previous step.
+
+Finally, we can run the new policies against your cloud environment in dryrun mode.
+This mode will only query the resources and apply the filters on the resources. Doing
+this allows you to assess the potential blast radius of a given policy change.
+
+Setting up the automated dryrun of policies is left as an exercise to the user-- this
+requires hosting your cloud authentication tokens inside of a CI system or hosting your
+own CI system and using Managed Service Identities (Azure) or Instance Profiles (AWS).
+
+It's important to verify that the results of the dryrun match your expectations. Custodian
+is a very powerful tool that will do exactly what you tell it to do! In this case, you should
+always "measure twice, cut once".
+
+
+Advanced Continuous Integration Tips
```

Comment:
lets move this to the bottom, I'd like the user to get through a complete working setup before they look at optimizing